lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -132,6 +147,14 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
                 <.remix_icon icon="dashboard-2-line" />
                 <span class="font-medium">See on Dashboard</span>
               </a>
+              <button class={"menu-item text-gray-500
+                #{if !session.memory_usage.runtime, do: "opacity-50 pointer-events-none"}"}
```

Comment:
Since this is a common use case we can extract `opacity-50 pointer-events-none` to `menu-item--disabled` :)